### PR TITLE
NO-SNOW Make schema evolution add columns idempotent

### PR DESCRIPTION
**Pull request by sfc-gh-xhuang**
NO-SNOW Make schema evolution add columns idempotent by using if not exists clause

### CODE REVIEW

**Review by sfc-gh-rcheng — APPROVED** [review]
lgtm! 
you'll need to run the formatter by running [./format.sh](https://github.com/snowflakedb/snowflake-kafka-connector#contributing-to-the-snowflake-kafka-connector) from repo root

File: src/main/java/com/snowflake/kafka/connector/internal/SnowflakeConnectionServiceV1.java [hunk 1]
```diff
@@ -488,7 +488,7 @@ public boolean hasSchemaEvolutionPermission(String tableName, String role) {
   public void appendColumnsToTable(String tableName, Map<String, String> columnToType) {
     checkConnection();
     InternalUtils.assertNotEmpty("tableName", tableName);
-    StringBuilder appendColumnQuery = new StringBuilder("alter table identifier(?) add column ");
+    StringBuilder appendColumnQuery = new StringBuilder("alter table identifier(?) add column if not exists ");
```
Comment (sfc-gh-tzhang) [inline src/main/java/com/snowflake/kafka/connector/internal/SnowflakeConnectionServiceV1.java:491]: Is this a GAed feature?

Comment (sfc-gh-xhuang) [reply src/main/java/com/snowflake/kafka/connector/internal/SnowflakeConnectionServiceV1.java:491]: yes

**Comment by sfc-gh-xhuang** [comment]
> lgtm! you'll need to run the formatter by running [./format.sh](https://github.com/snowflakedb/snowflake-kafka-connector#contributing-to-the-snowflake-kafka-connector) from repo root

Can I run that from github actions? Or I have to do it locally?

**Comment by sfc-gh-rcheng** [comment]
@sfc-gh-xhuang you'll need to do it locally and then commit the changes

**Pull request merged**